README.md

```diff
@@ -1,3 +1,24 @@
 # NLSProblems - Nonlinear Least Squares problems
 
-Uses NLSModels.jl.
+This package provides some optimization problems using the
+[https://github.com/JuliaSmoothOptimizers/NLPModels.jl](NLPModels.jl)
+interface. It is similar to to
+[https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl](OptimizationProblems.jl),
+except that it's specific for Nonlinear Least Squares, using the subtype
+of `AbstractNLSModel`s.
```

Comment:
Thanks. It would be enough to say "This package provides the nonlinear least-squares problems of Moré, Garbow and Hillstrom [REFERENCE] using the NLPModels.jl interface."